tests/lts_compatibility.py

```diff
@@ -93,6 +95,9 @@ def test_new_service(
     nodes_to_cycle = network.get_joined_nodes() if cycle_existing_nodes else []
     nodes_to_add_count = len(nodes_to_cycle) if cycle_existing_nodes else 1
 
+    # Pre-2.0 nodes require X509 time format
+    valid_from = str(infra.crypto.datetime_to_X509time(datetime.datetime.now()))
```

Comment:
This is surprising because the dynamic certificate validity range feature was added in 2.0?

Reply:
All those changes were prompted by test failures, so it's definitely required, but it may have been a 2.0-rcX network

Reply:
Also! The 2.0 checks are also run on 1.x networks in the LTS tests, so they need to support both.

Reply:
Oh I see, `valid_from` is automatically set as a ISO 8601 time string inside `network.trust_node()` if unset. Perhaps the comment should say "Test ASN1 UTC time string explicitly as default is ISO 8601"?

As discussed the LTS compatibility check should primarily check the compatibility with the latest 1.x so we should fix this.